pyzscaler/zia/admin_and_role_management.py

```diff
@@ -8,15 +8,13 @@ class AdminAndRoleManagementAPI(APIEndpoint):
     def add_user(self, name: str, login_name: str, email: str, password: str, **kwargs) -> Box:
```

Comment:
I'm not sure what was changed in this file, just looks like a blank line was removed somewhere. I think we can remove this from the pull request.